specification/monitor/resource-manager/Microsoft.Insights/preview/2023-03-15-preview/scheduledQueryRule_API.json

```diff
@@ -0,0 +1,853 @@
+      "properties": {
+        "actionGroups": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Action Group resource Ids to invoke when the alert fires."
+        },
+        "customProperties": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "description": "The dictionary of custom properties to include with the post operation. These data are appended to the alert payload."
+          },
+          "description": "The properties of an alert payload."
+        },
+        "actionProperties": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "description": "The dictionary of action properties to include with the post operation. This data customizes the actions."
+          },
+          "description": "The properties of an action properties."
+        }
+      }
+    },
+    "RuleResolveConfiguration": {
+      "type": "object",
+      "description": "TBD. Relevant only for rules of the kind LogAlert.",
+      "properties": {
+        "autoResolved": {
+          "type": "boolean",
+          "description": "The flag that indicates whether or not to auto resolve a fired alert."
+        },
+        "timeToResolve": {
+          "type": "string",
+          "format": "duration",
+          "description": "The duration a rule must evaluate as healthy before the fired alert is automatically resolved represented in ISO 8601 duration format."
+        }
+      }
+    },
+    "ScheduledQueryRuleProperties": {
+      "type": "object",
+      "properties": {
+        "createdWithApiVersion": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The api-version used when creating this alert rule"
+        },
+        "isLegacyLogAnalyticsRule": {
+          "type": "boolean",
+          "readOnly": true,
+          "description": "True if alert rule is legacy Log Analytic rule"
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the scheduled query rule."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the alert rule"
+        },
+        "severity": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Severity of the alert. Should be an integer between [0-4]. Value of 0 is severest. Relevant and required only for rules of the kind LogAlert.",
+          "enum": [
+            0,
+            1,
+            2,
+            3,
+            4
+          ],
+          "x-ms-enum": {
+            "name": "AlertSeverity",
+            "modelAsString": true
+          }
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "The flag which indicates whether this scheduled query rule is enabled. Value should be true or false"
+        },
```

Comment:
>}, [](http://example.com/codeflow?start=8&length=2)

Removing publicNetworkAccess is a breaking change. You will need to take this through breaking change review.

Reply:
publicNetworkAccess only exists in 2022-08-01-preview.
From the documentation - it sounds like I don't need to pass a review because both are preview versions. 
Is my understanding correct?

Reply:
@mentat9

Reply:
Breaks are allowed when they only break a prior preview version. I will remove the BreakingChangeReviewRequired label.